environments/production/app-processes.yml

```diff
@@ -63,6 +63,9 @@ celery_processes:
     saved_exports_queue:
       concurrency: 4
       max_tasks_per_child: 1
+    case_update_rule_queue:
```

Comment:
We'd want this on other envs right?

Reply:
Not sure.  For other environments the task will use the background_queue.  I'd be interested in seeing if this helps on production before rolling out the change to other environments, but I don't have strong opinions on this.